AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/asset_processor_gui_tests.py

```diff
@@ -370,7 +359,7 @@ def test_WindowsPlatforms_LaunchAP_LogReportsIdle(self, asset_processor, workspa
         # Launch Asset Processor and wait for it to go idle
         result, _ = asset_processor.gui_process()
         assert result, "AP GUI failed"
-        ap_idle.check_if_idle()
+        assert asset_processor.wait_for_idle(timeout=30)
```

Comment:
What does timeout=30 mean? can you explain why this was needed for the test conversion?

Reply:
The zip file is so AP doesn't immediately load all the files in the test folder, instead taking the zip and creating a folder with the file structure Levels/C#####/TestDependencies.prefab when it unzips. 

The timeout was the base timeout used in the ap_idle.check_if_idle() fixture, which is 30 seconds. This fixture is declared within the module and likely used by other tests, so I didn't touch the fixture itself; instead I updated the test to assert  asset_process.wait_for_idle - which is using the asset_processor_setup.fixture

It will return True if AP returns idle or False if AP reaches the 30 second timeout threshold.